RoboJackets/robocup-software
# Coach Node Publishes Static Obstacles

## Description
The Coach node now publishes two sets of static obstacles (physical walls of the goal as well as defense areas) to the respective topics, copying the functionality from the old gameplay node.

## Associated / Resolved Issue
Resolves [ClickUp card](https://app.clickup.com/t/346bh0r)

## Steps to Test
### Testing in Sim
1. Run sim. Should see red boxes around goal walls and defense areas.
2. Change from stop to play and vice-versa. Opponent's defense area size should change accurately.
3. Watch defender (robot 1). It has been currently set to oscillate at y=0.5.

**Expected result:** Robot 1 should avoid the defense area.

## Key Files to Review
Coach Node
 * `coach_node.cpp`
 * `coach_node.hpp`
 * Note: `defense.cpp` was modified to make testing simple and the effects of this PR obvious. If this is harmful for the future, can be easily changed. 

## Review Checklist
